index.rst

```diff
@@ -1,51 +1,517 @@
+In my initial implementation I will ignore the effect of variable seeing, and use only observations with comparable PSFs, but in order to make full use of all the data available it will have to be addressed in the future (see `Extension to variable seeing`_).
+Thus, for this initial investigation I will assume that the only effect that changes the shape of the PSF over a set of observations is DCR.
+
+:numref:`fig-subband_diagram` below illustrates the approach of this algorithm. Since DCR arises from the change in the index of refraction of the atmosphere across a filter bandpass, if we can build a model sky in smaller sub-bands the effect is greatly reduced.
+Further, if DCR is the only effect on the PSF, then we can construct these sub-band models with a small enough bandwidth such that the model to be the same for all observations regardless of airmass and parallactic angle, except for a bulk shift of the entire image.
+We only measure the combined image from all sub-bands, however, so those shifted sub-band images must be added together, which results in an apparent elongation of the PSF.
+
+.. figure:: /_static/DCR_subband_diagram.png
+   :name: fig-subband_diagram
+
+   A star observed at different parallactic and zenith angles may appear slightly elongated in the zenith direction, but this is due to DCR.
+   If the full filter were split into two sub-bands (here **A** and **B**), the star would appear round(er) but shifted slightly from the position measured across the full band.
+   If we assume that the 'true' image within each sub-band does not change, then the changing elongation in the **N** observed images along the zenith direction can be attributed to these shifts, which we can calculate with :eq:`eqn-refraction`.
+   
+Notation
+^^^^^^^^
+
+I will use matrix notation throughout this note, with vectors written as lower case letters and 2D matrices written as upper case letters.
+In this context, images are written as vectors, with all of the pixel values unwrapped.
+To emphasize this point, I have added arrows over the vectors, though these don't convey any additional meaning.
+Image data is written as :math:`\overrightarrow{s_i}`, where the subscript :math:`i` loops over the input images, while model data is written as :math:`\overrightarrow{y_\alpha}` with the subscript :math:`\alpha` looping over sub-bands.
+While it is often convenient for :math:`\overrightarrow{y_\alpha}` to have the same resolution and overall pixelization as :math:`\overrightarrow{s_i}`, in general even a non-gridded pixelization such as `HEALPix <https://healpix.jpl.nasa.gov>`_ could be used.
+
+The matrix :math:`B_{i\alpha}` encodes the transformation due to DCR of model plane :math:`\alpha` to image :math:`i`, and the reverse transformation is written as :math:`B_{\alpha i}^\star`.
+Since the sub-bands have a narrow bandwidth, the effect of DCR is a uniform shift of all pixels, so:
```

Comment:
How narrow do the sub-bands need to be for this approximation to hold?

Reply:
I've added a new section describing finite bandwidth effects, and how I deal with them.